### DISCUSSION TITLE
“Error updating branch: update failure”

### DISCUSSION THREAD

**Discussion by izorster** [renovatebot/renovate#18648] [Request Help] [answered]
Running renovate to keep dockerhub image alpine/git up-to-date:

renovate.json:
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "cargo": {
    "enabled": false
  }
}

Dockerfile:
# Copy app to slim image
FROM alpine/git:@sha256:a3e645585f160e0589529b8a3ca0fcfdba717ab93507de39ba5a96e207972927

# COPY --from=builder /build/target/release/pkg /usr/bin
COPY target/release/pkg /usr/bin

Error message:
... It does find a newer version/sha
{"name":"renovate","hostname":"runner-sc1djcun-project-339-concurrent-0lwsgl","pid":14,"level":20,"logContext":"wT10JYRibfMRT6JXznWfW","repository":"pro/image/pkg","digest":"sha256:1dc958bc2b4770e2686acc73e23b8050b0d1b9e22c128b6fa39f5283874ad6cf","msg":"Got docker digest","time":"2022-10-28T16:28:31.604Z","v":0}

... Cannot create a renovate branch/PR/MR for it

{"name":"renovate","hostname":"runner-sc1djcun-project-339-concurrent-0lwsgl","pid":14,"level":40,"logContext":"wT10JYRibfMRT6JXznWfW","repository":"pro/image/pkg","branch":"renovate/alpine-git-digest","msg":"Error updating branch: update failure","time":"2022-10-28T16:28:32.023Z","v":0}

**Comment by Gabriel-Ladzaretti** [comment, marked as answer]
Seems there is already an existing branch for it, maybe even a PR. 
I would check the commit history for that branch, and possibly delete it and run Renovate again.

**Reply by izorster** [reply]
I used a sha version that was previously updated by renovate.
